apps/api/src/app/rate-limiting/usecases/get-api-rate-limit-configuration/get-api-rate-limit-configuration.usecase.ts

```diff
@@ -0,0 +1,44 @@
+import { Injectable } from '@nestjs/common';
```

Comment:
We have hardcoded defaults for the rate limit configuration which can be overidden with environment variables.